### PR TITLE
Improved code quality greatly, fixed up (some) vars

### PR DESCRIPTION
**Pull request by StopFlaggingMeJesusChrist**
I've basically converted the entire project to use the
`code {

}` style. Uh, have fun?

Credits:
pius1212#9229
for helping me with a ton of vars
John.#9309
Also for helping me with vars
big bong theory#5018
Code style and var suggestions
and the Horion team, for making this amazing utility mod.

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
Does this fix the issue of "use unicode font" not changing anything?

**Comment by StopFlaggingMeJesusChrist** [comment]
Nah

File: Horion/Module/Modules/AutoTotem.cpp [hunk 1]
```diff
@@ -2,37 +2,32 @@
...
-	{
+	if (g_Data.getLocalPlayer() != nullptr)  {
 		C_ItemStack* i = g_Data.getLocalPlayer()->getEquippedTotem();
 
-		if (i->item == NULL && delay > 3) 
-		{
+		if (i->item == NULL && delay > 3)  {
 			C_PlayerInventoryProxy* supplies = g_Data.getLocalPlayer()->getSupplies();
-			C_Inventory* a = supplies->inventory;
+			C_Inventory* Item = supplies->inventory;
```
Comment (richardletshacks) [inline Horion/Module/Modules/AutoTotem.cpp]: lowercase please

File: Horion/Command/Commands/BindCommand.cpp [hunk 2]
```diff
@@ -33,16 +30,14 @@ bool BindCommand::execute(std::vector<std::string>* args)
...
-		for(int i = 0; i < 190; i++)
-		{
+		for(int i = 0; i < 190; i++) {
 			const char* haystack = KeyNames[i];
 			size_t len = strlen(needle) + 1;
 			char* haystackLowercase = new char[len];
 			for (int i = 0; i < len; i++)
 				haystackLowercase[i] = tolower(haystack[i]);
 
-			if(strcmp(needle, haystackLowercase) == 0)
```
Comment (richardletshacks) [inline Horion/Command/Commands/BindCommand.cpp]: Why not also give these variables better names?

File: Horion/Command/Commands/EjectCommand.cpp [hunk 3]
```diff
@@ -2,19 +2,16 @@
 
 
 
-EjectCommand::EjectCommand() : IMCCommand("eject", "Removes the cheat from the game.", "")
-{
+EjectCommand::EjectCommand() : IMCCommand("eject", "Removes Horion Utility Mod from the game.", "") {
```
Comment (richardletshacks) [inline Horion/Command/Commands/EjectCommand.cpp]: use "mod" here

File: Horion/Command/Commands/HelpCommand.cpp [hunk 4]
```diff
@@ -2,22 +2,19 @@
...
 }
 
-bool HelpCommand::execute(std::vector<std::string>* args)
-{
+bool HelpCommand::execute(std::vector<std::string>* args) {
 	g_Data.getGuiData()->displayClientMessageF("==========");
 	std::vector<IMCCommand*>* commandList = cmdMgr->getCommandList();
 	for (auto it = commandList->begin(); it != commandList->end(); ++it) {
-		IMCCommand* plump = *it;
+		IMCCommand* plump = *it; // TODO: change plump
```
Comment (richardletshacks) [inline Horion/Command/Commands/HelpCommand.cpp]: just change it to something like "command" or "cmd", idk

File: Horion/Command/Commands/HideCommand.cpp [hunk 5]
```diff
@@ -2,25 +2,20 @@
...
 
-bool HideCommand::execute(std::vector<std::string>* args)
-{
+bool HideCommand::execute(std::vector<std::string>* args) {
 	GameData::hide();
-	if (GameData::shouldHide())
-	{
-		clientMessageF("[%sHorion%s] %sCheat Hidden", GOLD, WHITE, GREEN);
+	if (GameData::shouldHide()) {
+		clientMessageF("[%sHorion%s] %sThe Horion Utility Mod has been hidden.", GOLD, WHITE, GREEN);
```
Comment (richardletshacks) [inline Horion/Command/Commands/HideCommand.cpp]: use "mod" here

File: Horion/Command/Commands/HideCommand.cpp [hunk 6]
```diff
@@ -2,25 +2,20 @@
...
 	}
-	else
-	{
-		clientMessageF("[%sHorion%s] %sCheat is now visible", GOLD, WHITE, GREEN);
+	else {
+		clientMessageF("[%sHorion%s] %sHorion Utility Mod is now visible.", GOLD, WHITE, GREEN);
```
Comment (richardletshacks) [inline Horion/Command/Commands/HideCommand.cpp]: use "mod" here

File: Horion/Loader.cpp [hunk 7]
```diff
@@ -108,7 +108,7 @@ DWORD WINAPI keyThread(LPVOID lpParam)
 
 			C_GuiData* guiData = g_Data.getClientInstance()->getGuiData();
 
-			if (guiData != nullptr) guiData->displayClientMessageF("%sHorion Client is not allowed in Valea Network.", RED);
+			if (guiData != nullptr) guiData->displayClientMessageF("%sHorion Utility Mod is not allowed in Valea Network.", RED);
```
Comment (richardletshacks) [inline Horion/Loader.cpp]: use "mod" here, or just remove it completely

**Review by richardletshacks — CHANGES_REQUESTED** [review]

File: Horion/Module/Modules/AutoTotem.cpp (hunk 1 above)
Comment (StopFlaggingMeJesusChrist) [reply Horion/Module/Modules/AutoTotem.cpp]: done

**Comment by ghost** [comment]
ok but lets not forget i basically done everyting

**Comment by StopFlaggingMeJesusChrist** [comment]
i credited you fucknugget

**Comment by ghost** [comment]
> 
> 
> i credited you fucknugget

hey thats racist you cant say that :pensive::pensive::pensive::pensive:

File: Horion/Command/Commands/HelpCommand.cpp (hunk 4 above)
Comment (StopFlaggingMeJesusChrist) [reply Horion/Command/Commands/HelpCommand.cpp]: gotcha richard 😳

File: Horion/Command/Commands/BindCommand.cpp (hunk 2 above)
Comment (StopFlaggingMeJesusChrist) [reply Horion/Command/Commands/BindCommand.cpp]: they're valid, see #developing channel in discord

**Review by richardletshacks — APPROVED** [review]

**Comment by StopFlaggingMeJesusChrist** [comment]
accept pr or i make every variable `LilPlump`.

**Comment by pius1212** [comment]
> ok but lets not forget i basically done everyting

*cough* nop m8

File: Horion/Loader.cpp [hunk 8]
```diff
@@ -144,7 +144,7 @@ DWORD WINAPI keyThread(LPVOID lpParam)
 		
 		Sleep(2); 
 	}
-	logF("Aight bro I'm boutta head out");
+	logF("Threads exiting...");
```
Comment (GodGamer029) [inline Horion/Loader.cpp]: I don't approve of this

File: Horion/Loader.cpp [hunk 9]
```diff
@@ -153,7 +153,7 @@ DWORD WINAPI keyThread(LPVOID lpParam)
 DWORD WINAPI injectorConnectionThread(LPVOID lpParam) {
 	logF("Injector Connection Thread started");
 
-	struct MemoryBoi {
+	struct loaderMemory {
```
Comment (GodGamer029) [inline Horion/Loader.cpp]: Struct-names start with a capital letter

File: Memory/Hooks.cpp [hunk 10]
```diff
@@ -269,15 +262,13 @@ __int64 Hooks::UIScene_render(C_UIScene* uiscene, __int64 screencontext)
...
 
-__int64 Hooks::RenderText(__int64 a1, C_MinecraftUIRenderContext* renderCtx)
-{
+__int64 Hooks::RenderText(__int64 a1, C_MinecraftUIRenderContext* renderCtx) {
 	static auto oText = g_Hooks.RenderTextHook->GetFastcall<__int64, __int64, C_MinecraftUIRenderContext*>();
 	C_GuiData* dat = g_Data.getClientInstance()->getGuiData();
 	DrawUtils::setCtx(renderCtx, dat);
 	if (GameData::shouldHide())
-		return oText(a1, renderCtx);
+		return oText(a1, renderCtx); {
```
Comment (GodGamer029) [inline Memory/Hooks.cpp]: what is this

File: Memory/Hooks.cpp [hunk 11]
```diff
@@ -445,9 +432,8 @@ __int64 Hooks::RenderText(__int64 a1, C_MinecraftUIRenderContext* renderCtx)
 			// Show disabled Modules?
 			//const bool extendedArraylist = g_Data.getLocalPlayer() == nullptr ? /* not ingame */ true : /* ingame */(GameData::canUseMoveKeys() ? false : true);
 			constexpr bool extendedArraylist = false;
-			std::set<IModuleContainer> modContainerList;
-			// Fill modContainerList with Modules
-			{
+			std::set<IModuleContainer> modContainerList; {
```
Comment (GodGamer029) [inline Memory/Hooks.cpp]: what the fuck grelom

File: Memory/Hooks.cpp [hunk 12]
```diff
@@ -918,13 +882,13 @@ int Hooks::BlockLegacy_getRenderLayer(C_BlockLegacy* a1)
 		if (strstr(text, "ore") == NULL)
 			if (strcmp(text, "lava") != NULL)
 				if (strcmp(text, "water") != NULL)
+					if (strcmp(text, "bedrock") != NULL) // lol added bedrock xray from suggestions
```
Comment (GodGamer029) [inline Memory/Hooks.cpp]: They pulled a sneaky on us

**Comment by ghost** [comment]
@GodGamer029 fixed your problems sir @StopFlaggingMeJesusChrist tell pius, john, racoon man or whoever you made do the "ugly code" that they do are have le retard 

uh and let me know if you want anything else done

**Comment by ghost** [comment]
also this repository is 9 commits behind horion master branch so might wanna fix that

**Comment by ghost** [comment]
good job guys `Horion-improvedcodequality\Horion\Command\Commands\HelpCommand.cpp(18,94):error C2065: 'plump': undeclared identifier`

this is a joke

**Comment by ghost** [comment]
ok it compiles with my edits epic epic epic pushed all that shit but @StopFlaggingMeJesusChrist you need to [see the more recent commits](https://github.com/horionclient/Horion/commits/master) and do your revision, thanks.

https://imgur.com/a/nvxwTJX

**Comment by GodGamer029** [comment]
you just reverted all commits in this pr

**Comment by GodGamer029** [comment]
this pr is a dumpster fire

**Comment by ghost** [comment]
i just [restored everything](https://github.com/StopFlaggingMeJesusChrist/Horion/commit/a85b6bdc91cf9e5f7654bec904f315a2da28c829) and made the changes but nvm

**Comment by GodGamer029** [comment]
im just gonna do this myself

**Comment by ghost** [comment]
> 
> 
> im just gonna do this myself

👍

**Comment by enescaglayan** [comment]
hours of labor are gone now

**Pull request closed without merging**